test/ruby/CMakeLists.txt

```diff
@@ -10,14 +10,18 @@ macro(add_ruby_test)
     set(target "test_ruby_${ARGV0}_${ARGV1}")
     add_test(
         NAME "${target}"
-        COMMAND "${RUBY_EXECUTABLE}" "-I${CMAKE_BINARY_DIR}/bindings/ruby" "${ARGV1}"
+        COMMAND "${RUBY_EXECUTABLE}" "-I${CMAKE_BINARY_DIR}/bindings/ruby" "-I${CMAKE_SOURCE_DIR}/test/ruby/libdnf/support" "${ARGV1}"
```

Comment:
I am sorry but there is no description in commit message what the changes does or why we need them.

Reply:
I've added a comment. Describing such changes in commit messages gets easily lost in history.